### DISCUSSION TITLE
Kornia Training API

### DISCUSSION THREAD

**Discussion by puffy310** [kornia/kornia#1512] [Questions] [answered]
I'm a little incompetent, can someone tell how to make a config? I'm trying to finetune MoblieViT
I'm doing it on google colab, so here is the notebook
https://colab.research.google.com/drive/1pzLX4eFMCQp6jarHSw1Az0fdoV6AQ2eX?usp=sharing

**Comment by edgarriba** [comment, marked as answer]
You can check this example: https://github.com/kornia/kornia/blob/master/examples/train/image_classifier/main.py#L12
Use hydra `ConfigStore` or create your own data structure with the needed config parameters

**Reply by puffy310** [reply]
Thank you so much :)
